docker-for-windows/wsl-tech-preview.md

```diff
@@ -25,7 +25,7 @@ Before you install Docker Desktop WSL 2 backend, you must complete the following
 
 # Download
 
-Download [Docker Desktop Edge 2.1.6.0](https://download.docker.com/win/edge/40807/Docker%20Desktop%20Installer.exe) or a later release.
+Download [Docker Desktop Edge 2.1.6.0](https://hub.docker.com/editions/community/docker-ce-desktop-windows/) or a later release.
```

Comment:
Thanks Stefan. Can we move the version number (2.1.6.0) out of the link?

Reply:
Good catch. I've updated the PR.